### PR TITLE
I18n for Page factory

### PR DESCRIPTION
**Pull request by clicman**
Guys! Its major feature. Please lookup ang give feedback.

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 1]
```diff
@@ -734,7 +696,7 @@ public String getTitle() {
 
     /**
      * Search for the given WebElement in page repository storage, that is being generated during preconditions to all
-     * tests. If element is found, return its title annotation. If nothing found, log debug message and return
+     * tests. If element is found, return its title annotation. If nothing found, LOG debug message and return
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: did you do refactor on 'log'-variable or stupid-ctrl-h?

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 2]
```diff
@@ -1157,7 +1128,7 @@ private static String getFieldTitle(Field field) {
 
         /**
          * Get title annotation of specified WebElement, and add it as a parameter to allure report results,
-         * with corresponding value. If there is no title annotation, log warning and exit
+         * with corresponding value. If there is no title annotation, LOG warning and exit
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: unnecessary rename to upper case again

**Comment by kosteman** [comment]
**successfully** tested on _mobile-inside_ branch

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 3]
```diff
@@ -1,52 +1,43 @@
 package ru.sbtqa.tag.pagefactory;
 
-import static java.lang.String.format;
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: Docne

**Pull request merged**